examples/decoding/plot_receptive_field.py

```diff
@@ -160,23 +160,26 @@
 
 
 ###############################################################################
-# Create and fit a stimulus reconstruction model (a.k.a. backward model)
-# ----------------------------------------------------------------------
+# Create and fit a stimulus reconstruction model
+# ----------------------------------------------
 #
-# We will now construct a model to find the linear relationship between the
-# speech signal and a time-delayed version of the EEG. This can be advantageous
-# as we exploit all of the available neural data in a multivariate context,
-# compared to the forward case which treats each M/EEG channel as an
-# independent feature. Therefore, low SNR stimuli such as speech will likely
-# have better backward models.
-
-# We use the same lags as in [1]_
+# We will now demonstrate another use case for the ReceptiveField module as we
+# try to predict the stimulus activity from the EEG data. This is known in the
+# literature as a decoding, or stimulus reconstruction model [1]_. A decoding
+# model aims to find the linear relationship between the speech signal and a
+# time-delayed version of the EEG. This can be useful as we exploit all of the
+# available neural data in a multivariate context, compared to the encoding
```

Comment:
I'm still a bit iffy on this wording. I think I see what you're saying now, but we should mention that what we gain in exploiting the multi-variate nature of the neural data, we lose in _not_ exploiting the multivariate nature of the stimulus.

What about `...decoding models might provide a better quality of fit (at the expense of not controlling for stimulus covariance), especially...`

Reply:
This makes me think that in the future we should write a tutorial specifically on the differences between encoding / decoding :-)

Reply:
Yeah I thought about including a "at the expense of interpretability" in this bit. But your suggestion is even better imo.